src/providers/MeetingProvider/index.tsx

```diff
@@ -29,8 +27,6 @@ interface Props {
   logLevel?: LogLevel;
```

Comment:
Since now you remove the `meetingSessionConfig` properties here, could we change this `Props` to extends the `MeetingManagerConfig` interface?

This Props is quite confusing before, and now it seems we are able to improve it.
@devalevenkatesh @ltrung How do you think?

Reply:
This is no longer valid after changes to the implementation.